server/journeys/referral/safer-custody/schemas.ts

```diff
@@ -1,11 +1,14 @@
 import z from 'zod'
+import { createSchema } from '../../../middleware/validationMiddleware'
 
 const ERROR_MSG = "Select if the Safer Custody team is already aware of this referral or not, or select 'I don't know"
 
-export const schema = z.object({
-  isSaferCustodyTeamInformed: z.enum(['YES', 'NO', 'DO_NOT_KNOW'] as const, {
-    message: ERROR_MSG,
-  }),
+export const YES_NO_ANSWER = z.enum(['YES', 'NO', 'DO_NOT_KNOW'] as const, {
```

Comment:
This enum could potentially be moved into validation utils? YES, NO, DO NOT KNOW must be quite common

Reply:
this zod shape contains an error message specific to this screen.. maybe refactor this later when we have a 2nd YES NO DO NOT KNOW answer later.